prow/scripts/cluster-integration/helpers/generate-ark-config.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -vx
+
+shout "Create a Secret for Ark"
+
+export SA_NAME=sa-kyma-backup-restore
+export SA_DISPLAY_NAME=sa-kyma-backup-restore
+export SECRET_FILE=sa-kyma-backup-restore
+export ROLE=roles/storage.objectAdmin
+
+CLOUD_CREDENTIALS_FILE_CONTENT_BASE64=$(gcloud iam service-accounts keys list --iam-account=$SA_NAME@"${GCLOUD_PROJECT_NAME}".iam.gserviceaccount.com --limit=1 --format="csv[no-heading](KEY_ID)")
```

Comment:
I agree with @strekm comment, service account should be defined in the preset and injected to the job. I think that you can add to the already existing preset (preset-kyma-backup-restore) service account file - of course name of the preset should be then changed.

Reply:
Ok I think I got it. It will be part of config.yaml in the commit